integration-cli/docker_cli_help_test.go

```diff
@@ -51,19 +52,14 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 		helpCmd := exec.Command(dockerBinary, "help")
 		helpCmd.Env = newEnvs
 		out, ec, err := runCommandWithOutput(helpCmd)
-		if err != nil || ec != 0 {
-			c.Fatalf("docker help should have worked\nout:%s\nec:%d", out, ec)
-		}
+		c.Assert(err, checker.IsNil, check.Commentf(out))
+		c.Assert(ec, checker.Equals, 0, check.Commentf("docker help should have worked\nout:%s\n", out))
```

Comment:
I tend to prefer asserting on `err` and ignoring `exitCode` (`ec` here) if we don't look for a specific exit code number.